etcdutl/snapshot/v3_snapshot.go

```diff
@@ -138,31 +139,42 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 		if v != nil {
 			ds.Version = v.String()
 		}
+		var kv mvccpb.KeyValue
 		c := tx.Cursor()
 		for next, _ := c.First(); next != nil; next, _ = c.Next() {
 			b := tx.Bucket(next)
-			if b == nil {
```

Comment:
Why remove? 

Sorry if that was already asked. Didn't follow previous review comments.

Reply:
I've removed it because that condition was impossible as we're iterating over buckets in db. I've put it back now.